pkgs/build-support/fetchgit/nix-prefetch-git

```diff
@@ -207,6 +207,9 @@ clone(){
     # Initialize the repository.
     init_remote "$url"
 
+    # Append extra config.
+    echo "${extraConfig}" >> .git/config
```

Comment:
This will break if `extraConfig` contains special characters like `"`. Perhaps a safer way is to write your string to a path in the store using `builtins.toFile` and then use "cat" to read that file instead of trying to "echo" the string.